CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/Extent.java

```diff
@@ -44,7 +44,7 @@
     private final float nradius;
     private final float lradius;
 
-    Extent(final float[] xyz, final float nradius, final float lradius) {
+    public Extent(final float[] xyz, final float nradius, final float lradius) {
```

Comment:
wondering whether it would be better making this protected instead in this instance (assuming it wasn't previously public for a reason)

Reply:
I had some issues with accessors but I missed this one I reckon.